src/modules/flow/iio/iio.json

```diff
@@ -421,8 +421,8 @@
            "name": "use_device_default_scale"
          },
          {
-             "data_type": "float",
-             "default": 0,
+             "data_type": "direction-vector",
+             "default": {},
```

Comment:
@yongli3 @barbieri I would recommend to use a single "rgb" option.
We have an option data type specific for this purpose.